index.md

```diff
@@ -33,16 +33,13 @@ to program: it teaches them to break down problems and solve them in terms of
 primitive operations that a computer can perform, and it teaches them to
 express those solutions using various programming languages. However, there is
 a lot more to software engineering than just programming. The existing
-curriculum never formally teaches students about the ubiquitous tooling-like
-source control, build systems, and testing infrastructure-that makes efficient
-development and maintenance of large software systems possible.
-
-Even tools that students are exposed to, like the POSIX shell, are not
-presented as things to be learned and understood. Instead, our courses treat
-them as obstacles to be brushed aside as quickly as possible so students can
-get back to the "real work" of programming. We give students pre-written build
-scripts and rote `g++` commands to memorize and don't expect them to know or
-care what's going on behind the scenes.
+curriculum does not spend much time focusing on the ubiquitous tooling -- like
```

Comment:
Bring back the Unicode em dashes please :)

Reply:
ASCII LYFE

Reply:
In all seriousness, MD should render to em-dash

Reply:
I didn't think that was part of standard Markdown, but if it works it works. Can we still get rid of the spaces though?